dojo/metrics/utils.py

```diff
@@ -531,13 +530,6 @@ def get_in_period_details(
     # Approach to age determination is db-engine dependent
     if "postgresql" in connection.settings_dict["ENGINE"]:
```

Comment:
If `postgresql` is now the only option now, `if` is not needed, and `age_detail = findings...` might be shifted left.